### ISSUE TITLE
1.19.63 released (connection to BedrockConnect not possible)

### ISSUE THREAD

**Issue by karl007** [Pugmatt/BedrockConnect#374]
My son updated the switch and now the BedrockConnect Server is not available anymore - "old server" (Server veraltet) is returned.

Can you please updgrade the BedrockConnect.

https://feedback.minecraft.net/hc/en-us/articles/13374653039501-Minecraft-1-19-63-Bedrock-

**Comment by karl007** [comment]
WOW, so fast, you are amazing - Thanks for your work man!

**Comment by karl007** [comment]
I tested it, but same error message. 
I removed the docker container and rebuild id, but no success.
Is there a way I can verify which Version I have running?

Anyone else with this new Version that can verify that this release work or not?

**Comment by karl007** [comment]
Tried it now with the explicit URL of the new Version in the Dockerfile (https://github.com/Pugmatt/BedrockConnect/releases/download/1.28/BedrockConnect-1.0-SNAPSHOT.jar), removed the container and rebuild it, but same issue - switch told the server is too old and becrock_connect log shows "null disonnected".

Any suggestions?

**Comment by Pugmatt** [comment]
Hmm, I ran a test to double-check, and it appeared to run fine on the latest game version, 1.19.63. Only thing I can imagine happening is the docker container is still using the old JAR for some reason. Or I would double-check your Minecraft game is on the release for 1.19.63, and not on beta/pre-release.

That being said, I agree that there should be a way to verify what release version is being used when booting up the server. I've re-uploaded the latest release assets with a new JAR that displays the release version on start-up, and will do that for future versions as well.

If you re-download the JAR / re-build the docker container, the BedrockConnect server log should start with the following line now:

```-= BedrockConnect (Release: 1.28) =-```

If it's not showing that line, and just showing ```-= BedrockConnect =-```, it's still using an older version.

**Comment by karl007** [comment]
Hi Pugmatt.

Thanks for this update, this helps a lot! I got "-= BedrockConnect (Release: 1.28) =-" when I tried on my local pc, but at the docker, after complete rebuild my container, only "-= BedrockConnect =-", so problem is the not refreshed docker container 🙁

It seems that stopping, removing and rebuild the container forced and without cache does not affect the docker-compose build.

So, for everyone that may have a similar problem if you have a docker-compose file, this works not to refresh the container:
```
docker stop bedrock_connect; docker rm bedrock_connect; docker build --force-rm --no-cache bedrock_connect; docker-compose up -d
```

But this works (but recreate the other docker-compose containers also).
```
docker-compose build --no-cache bedrock_connect && docker-compose up -d --force-recreate
```
